failed to compile due to magick_rust issue

first: thanks, cool project.

I've tried installing, and got `error[E0425]: cannot find value QuantumRange in module bindings`
which is https://github.com/nlfiedler/magick-rust/issues/40

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@ crate-type = ["cdylib"]      # Creates dynamic lib
 
 [dependencies]
 miniserde = "0.1"
-magick_rust = "0.15"
+magick_rust = {version="0.15", features=["disable-hdri"]}
 regex = "1.5.4"
 sha2 = "0.10"
 which = "4"

